@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -678,7 +679,7 @@ const getExistingDisks = (template: XoVmTemplate) =>
 
 const automaticNetworks = computed(() => networks.value.filter(network => network.other_config.automatic === 'true'))
 
-const getDefaultNetworks = (template?: XoVmTemplate) => {
+const getDefaultNetworks = (template?: XoVmTemplate): XoNetwork[] | [] => {
```

Comment:
Why adding an empty array?
`XoNetwork[]` Isn't that enough?
```suggestion
const getDefaultNetworks = (template?: XoVmTemplate): XoNetwork[] => {
```

Reply:
I think is because we have an early return on the next line